cmd/controller-bootstrap/command/apiutil.go

```diff
@@ -13,7 +13,177 @@
 
 package command
 
-// TODO: getServiceResources infers aws-sdk-go to fetch the service metadata and custom resource names
-func getServiceResources() error {
-	return nil
+import (
+	"errors"
+	"fmt"
+	"github.com/gertd/go-pluralize"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+
+	awssdkmodel "github.com/aws/aws-sdk-go/private/model/api"
+)
+
+type metaVars struct {
+	ServiceID           string
+	ServicePackageName  string
+	ServiceModelName    string
+	ServiceAbbreviation string
+	ServiceFullName     string
+	CRDNames            []string
+}
+
+// SDKHelper is a helper struct to work with the aws-sdk-go models and
+// API model loader
+type SDKHelper struct {
```

Comment:
nit: Please add a TODO to move this struct and it's corresponding methods inside aws-controllers-k8s/pkg directory